db/migrate/20251215161508_rename_answer_analyses_to_answer_topics.rb

```diff
@@ -0,0 +1,27 @@
+class RenameAnswerAnalysesToAnswerTopics < ActiveRecord::Migration[8.0]
+  class AnswerAnalysis::AnswerTopics < ApplicationRecord; end
+
+  def up
+    rename_table :answer_analyses, :answer_topics
```

Comment:
```suggestion
    rename_table :answer_analyses, :answer_analysis_topics
```

Might be better to have all these tables prefixed with answer_analysis